src/app/modules/auth/login.page.tsx

```diff
@@ -1,26 +1,40 @@
-import PasswordInput from "../../../atomic/mol.password-input";
+import Form from "../../../atomic/org.form/form.component";
+import { loginSchema, LoginData } from "../../../atomic/org.form/form.schemas";
+import {
+  TextFormFields,
+  PasswordFormFields,
+} from "../../../atomic/mol.input-fields";
```

Comment:
Poderia usar absolute import aqui para evitar fazer todo esse caminho que pode começar a ficar confuso, dá uma lida aqui:
https://medium.com/thefork/escaping-relative-import-paths-hell-in-javascript-9f258baeb15e